mypy/join.py

```diff
@@ -391,7 +391,17 @@ def combine_similar_callables(t: CallableType, s: CallableType) -> CallableType:
 
 
 def combine_arg_names(t: CallableType, s: CallableType) -> List[Optional[str]]:
-    # Note: assumes is_similar_type(t, s) is true
+    """Takes two callables and produces the meet of their argument names.
+
+    For example, if 't' had the signature (a: int, b: str, c: str) -> None
+    and 's' had the signature (a: int, b: str, z: str) -> None, the
+    join of their argument names would be ["a", "b", None].
+
+    This information is then used to compute more accurate joins for
```

Comment:
Meets and joins, right?

Reply:
I think it's just joins -- this pull request doesn't touch meet.py.

(iirc, the meet of two types 't' and 's' would be some type that's basically a valid subtype of both. I couldn't think of a way of creating a callable that would be a valid subtype of two callables with different arg names without doing something messy with keyword arguments, so didn't attempt to modify that file.)